views/index.hjs

```diff
@@ -28,6 +28,15 @@
 			<td class='hn-post nowrap'>
 				Hacker News
 			</td>
+			<td class='hn-mentions nowrap'>
```

Comment:
As an aside, we should add some padding to the titles of the columns because they looked squished right now. That's a separate issue though.